how_to_submit_a_pr_-_quick_guide.md

```diff
@@ -0,0 +1,101 @@
+8. **Create PR **with WIP prefix in the title, assign it to yourself.
+9. Start coding, **push code regularly** to allow code review during your work.
+10. When work is done and ready for review, **push the final code**.
+
+####3. Test your work and submit for review
+11. Go over the user stories and **check your work** at least once for each case. If this is a bug make sure to include in your testings the ***exact*** scenario of the bug, and to add this scenario to the user stories.
+12. **Code review yourself** first, then make sure travis return green (both with Coder and Tests).
+13. Add a **screenshot**.
+14. **When all done** 
+  15. remove the WIP from the PR title 
+  16. assign it to the Team Lead developer for review
+  17. add relevant label.
+
+Read [Pull Requests](https://www.thegizraway.com/pull_requests.html) for more information.
+
+
+## Coding standard 
+ 
+Make sure you know the coding standard required for the language, the coder should run on your files to confirm this.
+
+* [How to setup coding standard in PHPStorm](https://www.jetbrains.com/help/phpstorm/2016.2/configuring-code-style.html#d1056806e40).
+* [Drupal coding standard](https://www.drupal.org/docs/develop/standards).
+
+## Issue tracking and Bugs
+
+Use Github issue tracking to document your work and progress when necessary, especially when working on bugs. This will allow other developers to help, and stakeholders to track your progress.
+
+
+## Github Issue Flow
+
+
+Typically an issue is opened by the account manager, or by the QA team/ Customer, the basic rule of thumb is that an issue is closed by the person who opened it. 
+This mean you can close issues opened by you when they are done and deployed after validation, and other issue should be assign to the account manager.
+
+Account managers or Team leads are responsible to assign issues to other team members.
+
+
+## Focus and Follow Up issues
+
+During your work try to stay on track, for bugs if you feel a quick solution can be thrown before dipping into a proper solution, please consult with your team lead.
+Any indirect issue you find during your work should be translated to a new follow up issue, and documented in your GH issue.  
+make sure you don't have more than 2 or 3 missions in progress. Stay focused.
+
+
+## Automated Tests
+
+Check with your Team Lead if a test should written or write one if you have extra time out of your timebox.
+
+## How deal with a problem
+
+Read [How to solve a problem](https://www.thegizraway.com/how_to_solve_a_problem.html).
+
+
+## Checklist
+
+[ ] - Merge latest code from the master branch.    
+[ ] - Code review yourself.  
+[ ] - Comment on all functions and comment complicated part of your code, be generous.  
+[ ] - Security check your code.  
+[ ] - Add a screen-shot.
+[ ] - Test your code, using the user stories, Add screenshot to the user story results. 
+[ ] - Add a wiki entry when needed, or release protocols.
+[ ] - Assign for review.
```

Comment:
This checklist needs to be at `PULL_REQUEST_TEMPLATE.md` file in all our active projects.
Example in Danel: https://github.com/Gizra/Danel/pull/1351

@bricel Is there a way to implement that in all project at once?